### DISCUSSION TITLE
Implementing Slickgrid in React. Pagination displays dots over button.

### DISCUSSION THREAD

**Discussion by jsoques** [ghiscoding/slickgrid-universal#486] [Q&A] [answered]
Hello, I am trying to use Slickgrid Universal in React and I have been sort of successful with some hacking. I am not a CSS expert and was wondering if someone can explain why does the paging nav display dots over the buttons as show in this screenshot.
![Untitled](https://user-images.githubusercontent.com/12414216/133701952-d633ca25-65d3-4956-bc01-fafe4851086a.png)

Thanks

**Comment by ghiscoding** [comment, marked as answer]
These DOM elements are <li> so it's probably showing li bullets, you can maybe try this Stack Overflow [answer](https://stackoverflow.com/a/2743201/1212166) with `list-style-type: none;`, maybe something like
```css
.slick-pagination {
  list-style-type: none;
}
```

I don't have that problem and since you can't provide a way to replicate, so it's a bit hard for me to fix something that I can't replicate, you'll have to try a few things and see what works then possibly make a contribution.

**Reply by jsoques** [reply]
Thanks, I will try this ASAP. Yes the idea is once I get something working that is fairly easy to implement I will make a contribution. Thanks again.

**Reply by ghiscoding** [reply]
BTW which styling framework, if any, are you using? I think I never caught this problem because most framework (Bootstrap, Bulma) would already have the `list-style-type: none;` in their CSS hence the reason I never caught it. I can add that line to the areas where I know I'm using Lists, it's just couple of places.

**Reply by jsoques** [reply]
Material-UI

**Reply by ghiscoding** [reply]
ok have u had a chance to try the css line I wrote about `list-style-type: none;`?

**Reply by jsoques** [reply]
Yes, it worked. Thanks.

**Comment by ghiscoding** [comment]
@jsoques 
A fix was done in PR #486 and is now released under the new version [0.18.0](https://github.com/ghiscoding/slickgrid-universal/releases/tag/v0.18.0)

Thanks for the feedback

**Reply by jsoques** [reply]
Yes thanks. I will test.

**Reply by jsoques** [reply]
@ghiscoding I removed the `list-style-type: none;` and upgraded and it's working. Thanks.

**Comment by ghiscoding** [comment]
@jsoques I just released an official version of [Slickgrid-React](https://github.com/ghiscoding/slickgrid-react) v2.0.0 and also added a fully working [Live Demo](https://ghiscoding.github.io/slickgrid-react/) 🚀

**Reply by jsoques** [reply]
Great, thanks.
